README.md

```diff
@@ -1725,7 +1725,9 @@ A GitHub conflict is when people make changes to the same area or line in a file
 | [Back To Top](#contents) |
 
 - ### **S**
-
+    
+    
+  - [Sabir Khan](https://github.com/DroidMystic)
```

Comment:
Please add your name in line 1732, after Saadat Badgujar.
Let us know if you need any help with that. 🙂